packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -1230,6 +1245,42 @@
                 }
             ],
             "passive": true
+        },
+        {
+            "name": "cwlLiveTail_Start",
```

Comment:
i'm still not sure why we can't just upstream these to commons

Reply:
I was just following the documentation here: https://github.com/aws/aws-toolkit-vscode/blob/master/docs/telemetry.md. I did this in VSCode so my local builds would generate the telemetry objects and I could bundle adding the metrics as well as using them in the same PR. If I need to update commons instead I can send a change for that.

Reply:
Opened PR to upstream to Commons. https://github.com/aws/aws-toolkit-common/pull/916